test/service.test.js

```diff
@@ -0,0 +1,38 @@
+const fs = require('fs');
+
+
+test("files exist", () => {
+    if(!fs.existsSync("./test_data/test")){
+        fs.mkdirSync("./test_data/test")
+    }
+    
+    expect(fs.existsSync("./test_data/test")).toBe(true);
+})
+
+
+test("files exist", () => {
```

Comment:
Generally we want to name these semantically, including some context around what doesn't exist and what context it's failing in.